docs/issuing-assets/control-asset-access.mdx

```diff
@@ -34,8 +34,8 @@ One thing you must decide when issuing an asset is what to call it. An asset cod
 
 Learn about liquidity pool shares in the [Liquidity Pool Encyclopedia Entry](../encyclopedia/liquidity-on-stellar-sdex-liquidity-pools).
 
-- Alphanumeric 4-character maximum: Any characters from the set [a-z][A-Z][0-9] are allowed. The code can be shorter than 4 characters, but the trailing characters must all be empty.
-- Alphanumeric 12-character maximum: Any characters from the set [a-z][A-Z][0-9] are allowed. The code can be any number of characters from 5 to 12, but the trailing characters must all be empty.
+- Alphanumeric 4-character maximum: Any characters from the set [a-z][a-z][0-9] are allowed. The code can be shorter than 4 characters, but the trailing characters must all be empty.
```

Comment:
Why must it change the second a-z? It is supposed to be capitalized... a-z, A-Z, 0-9

Reply:
Hmm probably because it things there's a link or something in there. Try escaping the first [ in each block so `\[`

Reply:
`\[a-z\][a-z\][0-9\]` that _might_ work